packages/renderer-stage3d/src/openfl/display/_internal/Context3DGraphics.hx

```diff
@@ -33,7 +33,7 @@ import openfl.display._internal.stats.DrawCallContext;
 @SuppressWarnings("checkstyle:FieldDocComment")
 class Context3DGraphics
 {
-	private static var blankBitmapData = new BitmapData(1, 1, false, 0);
+	private static var blankBitmapData = new BitmapData(1, 1, false, 0xFFFFFFFF);
```

Comment:
Offsets are applied after multipliers, so to make color fill + color transform to work properly, we need base bitmap to be whole white and color/color transform should be applied as a multiplier combo.